### DISCUSSION TITLE
Adding colors to colors.js file

### DISCUSSION THREAD

**Discussion by SkanderBahrini** [tailwindlabs/tailwindcss#3847] [Ideas]
Hello tailwindcss community, i am writting this message in order to inform you that i decided to add some colors to the template of colors in the colors.js file. Hope you will accpet my contribution. Best regards.

**Comment by Strickland235** [comment]
Define a separate js file colors.js, that consists of the colors object and export it.

export const COLORS = {
  white: '#ffffff',
  black: '#000',
  // your colors
}
Now import COLORS from the above file - import {COLORS} from './<Path>/colors.js' in your respective file that uses the defined colors. And use it as shown below.

button: {
  color: COLORS.white,
  backgroundColor: COLORS.black
}


[imessageapp](https://www.imessageapp.biz/)
